### ISSUE TITLE
1st weekly update

### ISSUE THREAD

**Issue by afeld** [cuny-nytech/jan-teamC#3]
* What is your project going to be?
* What is your [MVP](http://en.m.wikipedia.org/wiki/Minimum_viable_product)?
    * What features are nice-to-have?
* What progress have you made?
* What are your tasks for the week?
* How are you going to split up the work?

You probably won't have perfect answers to all these questions yet, so feel free to have the in-team conversations here.

**Comment by sdjukic** [comment]
Hi Aidan.
* we are going for clone of dropbox meet twitter. Users will be able to upload files (text, audio, video) and post comments (feeds).
* At least upload/download of textual files. It would be nice to have streaming of videos and music.
* That is the progress. Definite direction in which we are going and division of tasks.
* For next week we should have landing page, with logo and amazon S3 upload capabilities.
* I am in charge of interfacing with AWS, Steven is working on art and user validation and verification, Raymond is project manager.

**Comment by afeld** [comment]
Please post updates for this week!

**Comment by sdjukic** [comment]
AWS interface is finished. Users can upload/download and remove their files from AWS. Server is enforcing limits on file uploads (client is doing this too, but don't feel comfortable just on client side enforcing) and total space user has in the cloud. We are feeling ambitious and will try solve streaming media files via PandaStream add-on for Heroku. Still to finish is users and post models/controllers. Hopefully we will have something running on Heroku by next week.

**Comment by afeld** [comment]
Very cool...but where is the code? Try to get things in front of each other in pull requests as often as possible to get feedback. 

Sounds like you are making progress - my overall advice is to stick to [the KISS principle](http://en.m.wikipedia.org/wiki/KISS_principle). Things like file upload limits are good, but not really meaningful if the end-to-end flow isn't working yet. Streaming media files is good, but you could probably get by with <audio> and <video> tags for now. Prioritize features by what proves the concept. 

What are next steps for everyone? Try using issues to keep track of who is working on what.

**Comment by sdjukic** [comment]
You are absolutely right. Code I am talking about is in my profile. I started it as a test and I wanted to integrate it into the codebase last, since it has some pieces that are not assigned to me in the division of labor, but I needed them anyway, like users (ultimately I'd upload app to heroku since we are using my AWS account, see below.)
Limiting came with using AWS. I am using my account and if we put it online I want to be in charge of space people are using for storing things, just being paranoid when putting something up on the web, since I will be the one paying for it.
We are using email thread to communicate progress and assignments, again you are right, we could've used issues for that.
